Utilities: Add a quotes database and 'fortunes' program

BenWiederhake opened the pull request:
I told you, we need a quotes ~~page~~ file! :D

This even comes with fancy formatting and a tasteful selection ready to go! :^)
And look at this wonderful integration: It even links directly to the quote on logbot! (However, it can't be opened in Browser yet, probably a TLS thing is missing I guess.)

![Bildschirmfoto_2021-03-08_23-05-13](https://user-images.githubusercontent.com/2690845/110388057-d1351780-8062-11eb-8903-1289c7fc6505.png)
![Bildschirmfoto_2021-03-08_23-05-29](https://user-images.githubusercontent.com/2690845/110388056-d1351780-8062-11eb-99d5-40271a4925aa.png)

CC everyone in #5698, especially @linusg

Even though the IRC channel is publicly logged, I would still think it is nice to first ask before you quote someone out of context, so either check whether they have agreed to do so in #5698, or just ask them in person. Thanks!

linusg commented on Userland/Utilities/fortunes.cpp:
```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```
you know... :wink:

<sub>\<kling\>I didn't write this!\</kling\></sub>

jcs commented:
IMHO, it should be called `fortune` to match the [historical Unix command](https://en.wikipedia.org/wiki/Fortune_(Unix)).

linusg reviewed (commented):
minor nitpick: can we call the `link` entry `url`?

BenWiederhake commented:
Changes since last push: ALL the things!

jcs commented on Userland/Utilities/fortune.cpp:
```diff
@@ -0,0 +1,156 @@
+    const auto& chosen_quote = quotes[i];
+    auto datetime = Core::DateTime::from_timestamp(chosen_quote.utc_time());
+
+    outln(); // Tasteful spacing
```
While tasteful, this seems non-standard

BenWiederhake replied:
Well, we don't try to implement POSIX fortune here; our storage format is also very non-standard. Is there any downside on the spacing? I find it makes it more readable, especially for larger quotes; see screenshots.

jcs replied:
It just seems unnecessary and arbitrary, but you're the author so do what you prefer.

(It's only more readable because there's so much color being puked onto the terminal everywhere \*ducks\*)

BenWiederhake commented:
The broken build seems to be due to a broken `Userland/Libraries/LibWeb/CSS/Parser/Token.h` on `master`.

BenWiederhake commented:
Changes since last push:
- Sneakily added a quote
- Rebased to build it on CI

@jcs: Thanks for the feedback! But I think I'll leave those extra newlines in for now, until someone implements `--fancy=off` or something. :)
